### DISCUSSION TITLE
Possibility of a Strife Port?

### DISCUSSION THREAD

**Discussion by juiicetan** [OnionUI/Onion#529] [Ideas]
A couple of years ago [an old strife port was rediscovered](https://forum.zdoom.org/viewtopic.php?f=231&t=68702). I was hoping I could maybe ask someone to try and port it to onion os? It runs off of PRBoom+ I believe, so in theory it could maybe work. Sorry if this sort of thing is looked down upon, I'm just trying to see if this is something people would be interested in.
![STRIFE00](https://user-images.githubusercontent.com/106464193/170855839-30352cf9-f897-4fab-ab96-173bdea42303.png)

**Comment by Sichroteph** [comment]
To my knowledge, PRBoom+ needs OpenGL  ?

**Reply by juiicetan** [reply]
Yeah I think you're right. I wonder if a chocolate strife port would be possible then? I've tried running strife through dosbox on th miyoo mini but the latency can be pretty horrible in larger areas. There's also not really a good way to switch weapons other than bringing up the on-screen keyboard for the number keys.
